packages/twenty-front/src/modules/favorites/hooks/useFavorites.ts

```diff
@@ -85,77 +78,97 @@ export const useFavorites = () => {
       favoriteRelationFieldMetadataItems,
       getObjectRecordIdentifierByNameSingular,
       false,
+      views,
+      objectMetadataItems,
     );
   }, [
     favoriteRelationFieldMetadataItems,
     getObjectRecordIdentifierByNameSingular,
     workspaceFavorites,
+    views,
+    objectMetadataItems,
+  ]);
+
+  const favoritesByFolder = useMemo(() => {
```

Comment:
let's avoid useMemo, they are useless 99.9% of the time:
- if you memo a non UI computation (most of the cases in your PR), you will save a negligeable amount of compute
- instead we should understand why we have re-renders triggering a computation twice (BUT also the UI recomputation which is far more expansive) and eliminate them